webDemo/package.json

```diff
@@ -28,9 +28,11 @@
     "react-native-shimmer-placeholder": "^2.0.8",
     "react-native-svg": "^12.1.0",
     "react-native-svg-transformer": "^0.14.3",
-    "react-native-ui-lib": "snapshot",
+    "react-native-ui-lib": "6.28.2-snapshot.2397",
```

Comment:
Are you sure you want to push it? I think you should keep using `snapshot` here

Reply:
nope, it just was so you can test locally :)